checker/src/javax/annotation/concurrent/GuardedBy.java

```diff
@@ -41,19 +44,23 @@
 // to the Lock Checker's @GuardedBy annotation) or on a method (in which case
 // it is a declaration annotation corresponding to the Lock Checker's @Holding
 // annotation).
+// It is preferred to use these Checker Framework annotations instead:
+//  org.checkerframework.checker.lock.qual.GuardedBy
+//  org.checkerframework.checker.lock.qual.Holding
 
+@TypeQualifier
+@SubtypeOf(GuardedByInaccessible.class)
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
-@Target({ ElementType.FIELD, ElementType.PARAMETER, ElementType.LOCAL_VARIABLE, ElementType.METHOD, ElementType.CONSTRUCTOR })
+@Target({ ElementType.METHOD, ElementType.CONSTRUCTOR, ElementType.FIELD, ElementType.PARAMETER, ElementType.LOCAL_VARIABLE, ElementType.TYPE_USE })
 @PreconditionAnnotation(qualifier = LockHeld.class)
-@PostconditionAnnotation(qualifier = LockHeld.class)
 public @interface GuardedBy {
     /**
-     * The Java expressions that need to be {@link LockHeld}.
+     * The Java value expressions that need to be held.
      *
      * @see <a
      *      href="http://types.cs.washington.edu/checker-framework/current/checkers-manual.html#java-expressions-as-arguments">Syntax
      *      of Java expressions</a>
      */
-    String[] value();
+    String[] value() default {};
```

Comment:
The default makes it easier to use from the Lock Checker - it is always safe to retrieve the value without checking for null.
String[] makes it easier to uniformly retrieve the value from the Lock Checker regardless of whether jcip, javax or our own GuardedBy are used. Users can continue to write jcip and javax annotations the old way (without curly braces).